### ISSUE TITLE
ABLM2 VST plugin can't connect to second plugin instance

### ISSUE THREAD

**Issue by akimaze** [LMMS/lmms#5992]
### Bug Summary

ABLM2 VST plugin can't connect to second instance. I tested 32 and 64 bit version.
Demo can be downloaded from: https://www.tbproaudio.de/products/ablm
It's working in FL Studio Demo even on linux (wine 6.6 - needs core fonts from winetricks).

#### Steps to reproduce

1. Add ABLM2 effect
2. Add second instance of ABLM2 
3. Change mode in second instance to Receive
4. -> See not connected in first instance

#### Expected behavior

ABLM2 should connect to first plugin instance.

#### Actual behavior

ABLM2 VST plugin can't connect.

#### Screenshot

![image](https://user-images.githubusercontent.com/63190361/115145984-d79cb300-a054-11eb-819b-807556e4ff8c.png)

#### Affected LMMS versions

Tested on stable 1.2 (windows) and current 1.3 master branch (linux).

#### Logs
<details>
  <summary>Logs from linux machine</summary>
<pre>
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Starting detached.
unique ID: TB55
RemotePlugin::DebugMessage: inputs: 2  output: 2
RemotePlugin::DebugMessage: creating editor
RemotePlugin::DebugMessage: editor successfully created
0108:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0108:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0110:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0110:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Starting detached.
unique ID: TB55
RemotePlugin::DebugMessage: inputs: 2  output: 2
RemotePlugin::DebugMessage: creating editor
RemotePlugin::DebugMessage: editor successfully created
</pre>
</details>

**Comment by DomClark** [comment]
LMMS runs each VST plugin in its own process (known as "bridging"), which may be causing issues for ABLM2. Since you managed to make it work in FL, can you try it again in FL, but set one of the instances to be bridged ([Settings -> Processing -> Make bridged](https://www.image-line.com/fl-studio-learning/fl-studio-online-manual/html/plugins/wrapper.htm#wrapper_processing))? If it behaves the same as in LMMS, that should point to bridging as the issue.

**Comment by akimaze** [comment]
@DomClark Yes, you are right. When I set one plugin to bridged in FL Studio Demo ABLM2 also can't connect.

**Comment by DomClark** [comment]
In that case, this is not something we can fix soon. We need to support either hosting multiple VST plugins in a single bridge process, or hosting VST plugins within the LMMS process itself.
